sdk/eventhub/azure-eventhub/stress/Dockerfile

```diff
@@ -1,8 +1,16 @@
 # syntax=docker/dockerfile:1
 
-FROM python:3.8-slim-buster
+ARG PYTHON_VERSION=3.8
+FROM mcr.microsoft.com/dotnet/runtime:5.0.9-focal
```

Comment:
I know I suggested this but it does look a little odd out of context. It might be worth at least a comment note about why we are using this image.

Reply:
I'll publish a base image for `python3.9` and `python3.10`, then rely on that. We'll be able to cut all these install steps anyway.